deploy/README.md

```diff
@@ -0,0 +1,90 @@
+# Running a Gaia Hub with docker-compose
+
+Please refer to the [Official Docs](https://docs.stacks.co/storage-hubs/overview.html) for the most up to date instructions.
+
+
+
+## MacOS for local testing/development
+
+The following assumes you have [Docker Installed](https://docs.docker.com/docker-for-mac/install/)
+* Recommended to also have [MacOS Homebrew](https://docs.brew.sh/Installation) installed
+* Use Homebrew to install jq  with `brew install jq`
+
+In your working directory:
+1. clone a copy of the [gaia repo](https://github.com/blockstack/gaia):
+    ```
+    $ git clone \
+          -b master \
+          --single-branch \
+          https://github.com/blockstack/gaia \
+      gaia
+    ```
+
+1. Change your cwd:
+    ```
+    $ cd gaia/docker
+    ```
+
+1. Create a copy of sample-disk.env and fill out the values:
+    ```
+    $ cp sample-disk.env disk.env
+    # Update the DOMAIN_NAME and CERTBOT_EMAIL variables to a domain which will host the Gaia Hub,
+    # and email you own to request an SSL cert
+    ```
+
+1. Start the server:
+    ```
+    $ docker-compose -f docker-compose-base.yaml -f docker-compose-disk.yaml --env-file disk.env up
+    ```
+
+1. Verify the server is responding locally:
+    ```
+    $ curl -s my-domain.com/hub_info | jq
```

Comment:
I could be misreading this section, but shouldn't the local macos dev endpoints all be localhost?

Reply:
I'm working on getting this running locally. Let's hold off until then.